sdks/python/apache_beam/testing/benchmarks/nexmark/nexmark_launcher.py

```diff
@@ -67,18 +67,20 @@
 import logging
 import sys
 import uuid
-
-from google.cloud import pubsub
+from time import sleep
```

Comment:
but what is the reason to import time instead of only needed methods here

Reply:
since you wouldn't have to import each function you want to use from time module. It's not a big deal but I believe calling time.sleep() and time.time() will concise enough.